apps/discovery-platform/dump.sql

```diff
@@ -102,6 +102,10 @@ CREATE TABLE public.registered_nodes (
     id character varying(255) NOT NULL,
     has_exit_node boolean DEFAULT false NOT NULL,
     chain_id integer NOT NULL,
+    hoprd_api_endpoint character varying(255) NOT NULL,
+    hoprd_api_port integer NOT NULL,
+    exit_node_pub_key character varying(255),
+    node_address character varying(255) NOT NULL,
```

Comment:
is this not the PeerId?

Reply:
This is the address we are going to use to fund, so it is the blockchain address where the node runner will receive the funds not the peer id